webpack.config.js

```diff
@@ -143,10 +143,14 @@ const config = {
   },
   devServer: {
     inline: true,
-    historyApiFallback: {rewrites: [
-      { from: /./, to: '/static/index.html' }
-    ]},
-    contentBase: path.join(__dirname, 'client', 'app'),
+    index: '/static/index.html',
+    historyApiFallback: {
+      index: '/static/index.html',
+      rewrites: [
+        { from: /.?/, to: '/static/index.html' },
+      ],
+    },
```

Comment:
Maybe this is not needed anymore with the `index` setting?

Reply:
no, I tried - it does not work without it. actually, I don't understand why, but it does not work.

Reply:
hm, well, I'm wrong, it works without this line. will remove it